eq-author/src/components/buttons/Button/RouteButton.js

```diff
@@ -16,8 +26,13 @@ class RouteButton extends React.Component {
     to: PropTypes.string.isRequired,
   };
   render() {
-    const { children, to, ...otherProps } = this.props;
-    return (
+    const { children, to, disabled, ...otherProps } = this.props;
+
+    return disabled ? (
+      <DisabledLink disabled {...otherProps}>
+        {children}
+      </DisabledLink>
+    ) : (
```

Comment:
You could do,
```
    if (disabled) {
      return (
        <DisabledLink disabled {...otherProps}>
          {children}
        </DisabledLink>
      );
    }
    return (
      <StyledRouteLink {...otherProps} to={to}>
        {children}
      </StyledRouteLink>
    );
```
to make it a bit more readable, but either is fine. Its up to you :)

Reply:
That makes sense. It is more readable and, after looking around the codebase, it seems to be the pattern we follow.